Rules/Strings.resx

```diff
@@ -810,6 +810,18 @@
   <data name="UseCompatibleCmdletsError" xml:space="preserve">
     <value>'{0}' is not compatible with PowerShell edition '{1}', version '{2}' and OS '{3}'</value>
   </data>
+  <data name="AvoidOverwritingBuiltInCmdletsName" xml:space="preserve">
+    <value>AvoidOverwritingBuiltInCmdlets</value>
+  </data>
+  <data name="AvoidOverwritingBuiltInCmdletsCommonName" xml:space="preserve">
+    <value>Avoid overwriting built in cmdlets</value>
+  </data>
+  <data name="AvoidOverwritingBuiltInCmdletsDescription" xml:space="preserve">
+    <value>Do not overwrite the definition of cmdlets that are included with PowerShell</value>
```

Comment:
Because each diagnostic record warning will be for each cmdlet violation separately
```suggestion
    <value>Do not overwrite the definition of  a cmdlets that is included with PowerShell</value>
```

Reply:
That change should be
```<value>Do not overwrite the definition of a cmdlet that is included with PowerShell</value>```

Reply:
Good catch, I forgot to update that as well.  Fixed now.